modules/raw_data_collection/pprmeta_collect_data.nf

```diff
@@ -2,13 +2,18 @@ process pprmeta_collect_data {
     publishDir "${params.output}/${name}/raw_data", mode: 'copy', pattern: "pprmeta_results_${name}.tar.gz"
     label 'ubuntu'
     input:
-        tuple val(name), file(output_lists)
+        tuple val(name), path(output_lists)
     output:
-        tuple val(name), file("pprmeta_results_${name}.tar.gz")
+        tuple val(name), path("pprmeta_results_${name}.tar.gz")
     script:
         """
         cat ${output_lists} | head -1 > ${name}_overview.txt
         tail -q -n+2 ${output_lists} >> ${name}_overview.txt
         tar -czf pprmeta_results_${name}.tar.gz ${name}_overview.txt
         """
+    stub:
```

Comment:
in this cases i would create a dummy result file in ppr meta process and don't add "stub:" here so the dry run will run the actual parser script